database/incremental-scripts/add-isArchived-field-db.sql

```diff
@@ -0,0 +1,131 @@
+ALTER TABLE `civichq`.`Apps` ADD COLUMN `IsArchived` TINYINT(1) NULL DEFAULT 1 AFTER `IsMaster`;
```

Comment:
@paul-raetchi should the default value be 0 for apps? they should be not archived by default :smile: